contracts/liquidity/claimer/src/state.rs

```diff
@@ -0,0 +1,14 @@
+use cw_storage_plus::{Item, Map};
+use euclid::msgs::claimer::{Claim, State};
+
+pub const STATE: Item<State> = Item::new("state");
+
+pub const CLAIMS: Map<u128, Claim> = Map::new("claims");
```

Comment:
What if we create a struct that contains `Claim`, `Sender`, and `User` as its elements, and use that struct as the value of `CLAIMS`. Would that be enough to no longer need `SENDER_CLAIMS` and `USER_CLAIMS`?